modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -142,7 +176,7 @@ public void join() throws InterruptedException {
 
                 LOG.error("Error when completing the copier", cause);
 
-                if (!isOk()) {
+                if (isOk()) {
```

Comment:
What?

Reply:
This is how it works in `LocalSnapshotCopier`. Otherwise, if no error gets registered from elsewhere, we will never register any error; but, if some error already happened somewhere, we will overwrite it. It seems better to only leave the first error.

Reply:
Ok, I completely misunderstood this construction. I apologize